provider/grpc.go

```diff
@@ -385,3 +402,13 @@ func (g *GRPCClient) GetOrderStatusStream(ctx context.Context, market, ownerAddr
 
 	return connections2.GRPCStream[pb.GetOrderStatusStreamResponse](stream, market), nil
 }
+
+// GetRecentBlockHashStream subscribes to a stream for getting recent block hash.
+func (g *GRPCClient) GetRecentBlockHashStream(ctx context.Context) (connections2.Streamer[*pb.GetRecentBlockHashResponse], error) {
+	stream, err := g.apiClient.GetRecentBlockHashStream(ctx, &pb.GetRecentBlockHashRequest{})
+	if err != nil {
+		return nil, err
+	}
+
+	return connections2.GRPCStream[pb.GetRecentBlockHashResponse](stream, ""), nil
```

Comment:
can you also clean up the aliases in this file?

Reply:
done